docs/runtimes/micronaut.md

```diff
@@ -0,0 +1,152 @@
+# Using Micronaut with NubesGen
+
+This documentation is for running Micronaut applications with NubesGen. There are two other options that might interest you:
+
+- You can also use [Java with NubesGen](java/), which is similar to Micronaut, but does not use the specific Micronaut application properties.
```

Comment:
`which is similar to Micronaut`

`Micronaut` cannot be used as a noun according the brand guidelines. If we are going to keep this sentence we should rewrite the sentence.